analysis_driver/dataset.py

```diff
@@ -162,7 +162,14 @@ def __init__(self, name, path, most_recent_proc=None):
         self.path = path
 
     def _is_ready(self):
-        return os.path.isfile(os.path.join(self.path, 'RTAComplete.txt'))
+        return True
+
+    def is_sequencing(self):
+        # Assume the run has started and not finished if the status is 'RunStarted' or if it hasn't yet appeared in the LIMS
+        if not clarity.get_run(self.name):
+            self.warning('Run %s not found in the LIMS', self.name)
+            return True
+        return clarity.get_run(self.name).udf.get('Run Status') == 'RunStarted'
```

Comment:
I just realised that the Run object in the genologics API will get cached and that the UDF `'Run Status'` will never change.
We should explicitly store and update the lims run object and update it at each call of `is_sequencing`
```python
@property
def lims_run(self):
    if not self._lims_run:
        self._lims_run = clarity.get_run(self.name)
    return self._lims_run

def is_sequencing(self):
    if not self.lims_run:
        self.warning('Run %s not found in the LIMS', self.name)
        return True
    # force the update of the LIMS object
    self.lims_run.get(force=True)
    return self.lims_run.udf.get('Run Status') == 'RunStarted'
```